content/en/docs/tutorials/hello-minikube.md

```diff
@@ -60,7 +60,7 @@ Now, switch back to the terminal where you ran `minikube start`.
 The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser.
 You can create Kubernetes resources on the dashboard such as Deployment and Service.
 
-If you are running in an environment as root, see [Open Dashboard with URL](#open-dashboard-with-url).
+If you are running in an environment as root, see the "URL copy and paste" tab.
```

Comment:
👋 @shurup 
Does the in-page link, `#dashboard-1` link to the second tab?

Reply:
@kbhawkey thanks for this clarification! This was what I actually started from. However, such a link didn't work in my browser (Firefox): nothing happens at all when I click on the `#dashboard-1` anchor. So I decided it was not an option. Perhaps I'm doing something wrong?

Reply:
```suggestion
To find out how to view the web dashboard in your browser, see the "URL copy and paste" tab.
```

Reply:
@sftim I think such a change might lead a bit away from the original point. This tab also implies viewing the dashboard in the browser; it just assumes it will be launched right from a currently open terminal. To avoid it — specifically for users executing the `minikube` commands as a root — this sentence suggests another option when your terminal will only display the URL, which you can later use in your browser (launched before as a non-root user).

To make it more clear, I suggest the following:
```suggestion
Since this command will launch a browser under your current terminal user, you should not run it as root. Alternatively, you can get a URL in the terminal and use it in a browser to open the web dashboard. See the "URL copy and paste" tab for details about this method.
```

What do you think?

Reply:
Should people be running _any_ of this tutorial as root?

Reply:
Valid thought!

Here's my new suggestion then — I think it's a bit more precise:
```suggestion
To find out how to avoid directly invoking the browser from the terminal and get a URL for the web dashboard, see the "URL copy and paste" tab.
```